packages/core/src/languages/solidity/predeploys.ts

```diff
@@ -175,21 +221,39 @@ export const deploySphinxSystem = async (
   for (const relayer of relayers) {
     const chainId = (await provider.getNetwork()).chainId
 
+    const currentDripVersion = fetchDripVersionForNetwork(chainId)
+    const baseDripName = `sphinx_fund_${relayer}`
+    const dripName =
+      baseDripName + (currentDripVersion > 0 ? `_${currentDripVersion}` : '')
+
+    // Cancel any out of date drips
+    await cancelPreviousDripVersions(
+      Drippie,
+      provider,
+      owner,
+      executionMode,
+      baseDripName,
+      currentDripVersion,
+      logger
+    )
+
     const reentrant = false
     const interval = 30
     const dripcheck = getCheckBalanceLowAddress()
     const checkparams = AbiCoder.defaultAbiCoder().encode(
       ['address', 'uint256'],
-      [relayer, ethers.parseEther(fetchDripSizeForNetwork(chainId))]
+      [relayer, ethers.parseEther(fetchDripSizeForNetwork(chainId)) * BigInt(5)]
```

Comment:
Here we're setting the maximum balance allowed in the recipient wallet. So if they have a balance greater than this, and you attempt to trigger another drip, it will fail due to them having a large balance. 

Initially, I just used a limit equal to 1 * dripSize, and this is how most of the networks are set up. However, it's possible we might have to drip multiple times to get enough funds to execute a very large deployment. This hasn't been an issue so far, so I haven't actually redeployed on all of the networks to adjust this. But I will if we run into problems related to this.